src/coins/coins.int.spec.ts

```diff
@@ -23,7 +23,7 @@ describe.concurrent('Coin logo test', { timeout: 30_000 }, () => {
     ...Object.values(wrappedTokens).map((token: StaticToken) => token.logoURI!)
   )
 
-  test.each([...new Set(allImages)])(
+  test.each(Array.from(new Set(allImages)))(
```

Comment:
Set does not support iteration on node 16

Reply:
the above complain is from my complier

Reply:
Why do you use node 16? 
If not present, feel free to add an `.nvmrc` and set it to 20 or so. 16 is very outdated

Reply:
My fault, the vscode compiler itself is wrong